packages/enzyme-example-mocha/.eslintrc

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "airbnb",
+  "root": true,
+  "env": {
+    "node": true,
+    "mocha": true
+  },
+  "globals": {
+    "expect": false
```

Comment:
instead of this, we should `import { expect } from 'chai';` - i didn't think mocha provides a global expect, but either way, all our examples are using chai